### ISSUE TITLE
ansible_service_mgr fact for sysvinit incorrect in ansible 2.4.0+ and devel

### ISSUE THREAD

**Issue by toke** [ansible/ansible#32068]
The issue #30753 should be fixed and the issue is closed which is not the case.

Still 2.4.1-0.2.beta2 is affected as is devel. So it won't make it into 2.4.1?

I suggest reopening #30753 and closing this one.

##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
Ansible fact / variable `ansible_service_mgr` 

##### ANSIBLE VERSION
```
ansible 2.4.0.0
ansible 2.5.0.0
ansible devel
```

##### CONFIGURATION
does not matter

##### OS / ENVIRONMENT
Target Debian Linux 7.11 (possible other Systems with sysvinit)

##### SUMMARY
Prior to ansible 2.4.0.0 ansible_service_mgr contained the value `sysvinit`, after the update to 2.4.0.0 ansible falls back to `service`. I would call it "alternative fact" right now, at least some roles fail due to this behaviour.

##### STEPS TO REPRODUCE
Outputting the fact ansible_service_mgr for a host running sysvinit (for example Debian 7.11 with sysvinit)

```
ansible -i host.example.com, -m setup  host.example.com | grep ansible_service_mgr
```

##### EXPECTED RESULTS
```
"ansible_service_mgr": "sysvinit",
```

##### ACTUAL RESULTS
```
"ansible_service_mgr": "service",
```

**Comment by toke** [comment]
Original Ticket has a testcase…

**Comment by toke** [comment]
Closed since #30753 was reopened
